docs/core/tutorials/publishing-with-visual-studio.md

```diff
@@ -1,49 +1,67 @@
+In [Build a Hello World application with .NET Core in Visual Studio](with-visual-studio.md), you built a Hello World console application. In [Debug your Hello World application with Visual Studio](debugging-with-visual-studio.md), you tested it using the Visual Studio debugger. Now that you're sure that it works as expected, you can publish it so that other users can run it. Publishing creates the set of files that are needed to run your application. To then deploy the files, copy them to the target machine.
 
-To publish and run your application:
+## Publish the app
 
 1. Make sure that Visual Studio is building the Release version of your application. If necessary, change the build configuration setting on the toolbar from **Debug** to **Release**.
 
    ![Visual Studio toolbar with release build selected](media/publishing-with-visual-studio/visual-studio-toolbar-release.png)
 
-1. Right-click on the **HelloWorld** project (not the HelloWorld solution) and select **Publish** from the menu. You can also select **Publish HelloWorld** from the main Visual Studio **Build** menu.
+1. Right-click on the **HelloWorld** project (not the HelloWorld solution) and select **Publish** from the menu. (You can also select **Publish HelloWorld** from the main **Build** menu.)
 
    ![Visual Studio Publish context menu](media/publishing-with-visual-studio/publish-context-menu.png)
+   
+1. On the **Pick a publish target** page, select **Folder**, and then select **Create Profile**.
 
-   ![Visual Studio Publish window](media/publishing-with-visual-studio/publish-settings-window.png)
+   ![Pick a publish target in Visual Studio](media/publishing-with-visual-studio/pick-publish-target.png)
+   
+1. On the **Publish** page, select **Publish**.
 
-1. Open a console window. For example in the **Type here to search** text box in the Windows taskbar, enter `Command Prompt` (or `cmd` for short), and open a console window by either selecting the **Command Prompt** desktop app or pressing Enter if it's selected in the search results.
+   ![Visual Studio Publish window](media/publishing-with-visual-studio/publish-page.png)
+   
+## Inspect the files
 
-1. Navigate to the published application in the `bin\release\PublishOutput` subdirectory of your application's project directory. As the following figure shows, the published output includes the following four files:
+The publishing process creates a framework-dependent deployment, which is a type of deployment where the published application runs on any platform supported by .NET Core with .NET Core installed on the system. Users can run the published app by issuing the `dotnet HelloWorld.dll` command from a command prompt.
```

Comment:
or just run helloworld.exe, according to the following list of files

Reply:
Good catch.